dbtc/client/cloud/base.py

```diff
@@ -77,10 +59,28 @@ def _header_property(self):
 
         return 'api_key'
 
+    def _clone_resource(self, resource: str, **kwargs):
```

Comment:
@dpguthrie this is very cool! Is there a means I could use to subset which fields from the base object are used for replication? It seems like this is the intent of `create_args`, but I think a field subset being passed there would need to affect the `payload` in order for that modification to occur.

Context: When trying to replicate a job, I'm not able to use fields such as `deferring_job_definition_id` on the CREATE JOB endpoint. The process needs to first create the job with fewer fields, and then add the `deferring_job_defintion_id` in an update.

Reply:
Ya, we need another argument here for that.  Good call!  Who knows this private method may be a bit overkill for the things we need to do as well as thinking about what resources actually need to be cloned.

Reply:
Per conversation offline, noting that we agreed this isn't something to pursue further for now. There will be too much situational variation in what's needed to replicate resources, so we should stick to the CRUD operations and adjust as-needed for other use cases.